requirements.md

```diff
@@ -18,8 +18,8 @@
 
 * **platform-independent** (usable on embedded, not just servers+desktops+mobile)
 
-* **composable** (pick the crypto suite and features you need, without incurring overwhelming dependencies for stuff you don’t)
+* **composable** (pick the crypto suite and features you need, without incurring overwhelming dependencies for stuff you don’t) {this would seemingly require some form of negotiation, similar to what happens in TLS.   What if the sender decided to cipher the message with SecretSquirrelLevel10 algo and the recipient cant handle that algo for any number of reasons (export controls, etc)?}
```

Comment:
This would be a good comment to raise in the issue, but I don't think we need to add it to the spec docs.